test/jest/config.js

```diff
@@ -0,0 +1,26 @@
+const path = require('path');
+
+module.exports = {
+  collectCoverageFrom: [
+    'ui_framework/components/**/*.js',
+    // Seems to be a bug with jest or micromatch, in which the above glob
+    // doesn't match subsequent levels of directories, making this glob necessary.
+    'ui_framework/components/**/**/*.js',
+    '!ui_framework/components/index.js',
+    '!ui_framework/components/**/*/index.js',
+  ],
+  coverageDirectory: '<rootDir>/target/test_coverage',
+  coverageReporters: ['html'],
+  moduleFileExtensions: ['jsx', 'js', 'json'],
+  testPathIgnorePatterns: [
+    '<rootDir>[/\\\\](bin|config|docs|maps|node_modules|optimize|scripts|src|style_guides|tasks|test|utilities|webpackShims)[/\\\\]',
```

Comment:
Let's add `build` to the list of ignored directories (otherwise we'll see a ton of errors in the terminal):

```
'<rootDir>[/\\\\](bin|build|config|docs|maps|node_modules|optimize|scripts|src|style_guides|tasks|test|utilities|webpackShims)[/\\\\]',
```